<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_producto</class>
 <widget class="QWidget" name="form_producto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulario de Producto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(140, 198, 63);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Informacion adicional : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guardar">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_categoria">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <item>
    <property name="text">
     <string>Libreria</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Almacen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autopartes</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_limpiar">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Lote : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lne_lote">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>150</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_id_codigo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>__</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unidad : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_unidad">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
   <item>
    <property name="text">
     <string>Kg</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unidad</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Litro</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="text_info">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>611</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 236);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>651</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Nombre : </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lne_nombre">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Descripcion :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lne_descripcion">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Codigo interno:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lne_codigo_interno">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Fecha vencimiento :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="2">
     <widget class="QDateEdit" name="dte_fec">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Codigo producto:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QLineEdit" name="lne_codigo_producto">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Marca:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lne_marca">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
